jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -28,24 +30,72 @@ class Catalogs(PluginTemplateMixin, ViewerSelectMixin):
     template_file = __file__, "catalogs.vue"
     catalog_items = List([]).tag(sync=True)
     catalog_selected = Unicode("").tag(sync=True)
+    from_file = Unicode().tag(sync=True)
+    from_file_message = Unicode().tag(sync=True)
     results_available = Bool(False).tag(sync=True)
     number_of_results = Int(0).tag(sync=True)
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        # TODO: Add more catalogs.
```

Comment:
This TODO is still relevant despite this patch, why is it removed?

Reply:
we have tickets/issues to do this, I didn't think it needed an inline comment, but I can revert if you'd like

Reply:
Hmm ok.